src/suvela/Assignment24.java

```diff
@@ -0,0 +1,28 @@
+/*Assignment - 24 : 22nd Nov'2020 
+
+program 1: sum of all digits from a given string  
+input: te1ch9kj8 
+output:= 18*/
+package suvela;
+
+public class Assignment24 {
+	void getSumOfDigits(String input) {
+		int i=0;
+		int sum=0;
+		for(int index=0;index<input.length();index++) {
+			char ch=input.charAt(index);
+			boolean flag=Character.isDigit(ch);
+			if(flag) {
+				i = Character.getNumericValue(ch);
```

Comment:
no need to define extra variable "i" as you can directly do add operation on sum